tests/test_row_column.py

```diff
@@ -137,135 +135,127 @@ def test_modify_column(client, table):
+#     resp = client.put(
+#         url_for('csv.batch_modify_label', csv_id=table.id),
+#         json={'changes': [{'context': 'column', 'index': 2, 'label': 'group'}]}
+#     )
+#     assert resp.status_code == 200
+#     assert resp.json == {
+#         'column_header': 'meta',
+#         'column_index': 2,
+#         'column_type': 'group',
+#         'data_column_index': None,
+#         'data_variance': None,
+#         'missing_percent': None
+#     }
+
+#     resp = client.get(
+#         url_for('csv.get_column', csv_id=table.id, column_index=1)
+#     )
+#     assert resp.status_code == 200
+#     assert resp.json == {
+#         'column_header': 'group',
+#         'column_index': 1,
+#         'column_type': 'metadata',
+#         'data_column_index': None,
+#         'data_variance': None,
+#         'missing_percent': None
+#     }
+
+
+# def test_modify_csv_file_header_index(client):
+#     table = """
+# ,,,
+# row1,g,0.5,2.0
+# row2,g,1.5,0
+# id,g,header1,header2
+# """
+#     csv_file = csv_file_schema.load({
+#         'table': table,
+#         'name': 'test_csv_file.csv'
+#     })
+#     db.session.add(csv_file)
+#     db.session.commit()
+
+#     csv_id = csv_file.id
+#     resp = client.put(
+#         url_for('csv.batch_modify_label', csv_id=csv_id),
+#         json={'changes': [{'context': 'row', 'index': 0, 'label': 'masked'}]}
+#     )
+#     assert resp.status_code == 200
+
+#     resp = client.put(
+#         url_for('csv.batch_modify_label', csv_id=csv_id),
+#         json={'changes': [{'context': 'row', 'index': 3, 'label': 'header'}]}
+#     )
+#     assert resp.status_code == 200
+#     assert (CSVFile.query.get(csv_id).measurement_table.columns == ['header1', 'header2']).all()
+
+
+# def test_modify_csv_file_key_index(client):
+#     table = """
+# header1,group,id,header2,header3,extra
+# 0,a,row1,0.5,2.0,
+# 0,b,row2,1.5,0,
+# """
+#     csv_file = csv_file_schema.load({
+#         'table': table,
+#         'name': 'test_csv_file.csv'
+#     })
+#     db.session.add(csv_file)
+#     db.session.commit()
+
+#     csv_id = csv_file.id
+#     resp = client.put(
+#         url_for('csv.batch_modify_label', csv_id=csv_id),
+#         json={'changes': [{'context': 'column', 'index': 2, 'label': 'key'}]}
+#     )
+#     assert resp.status_code == 200
+
+#     resp = client.put(
+#         url_for('csv.batch_modify_label', csv_id=csv_id),
+#         json={'changes': [{'context': 'column', 'index': 5, 'label': 'masked'}]}
+#     )
+#     assert resp.status_code == 200
+#     assert (CSVFile.query.get(csv_id).measurement_table.columns == ['header2', 'header3']).all()
+
+#     resp = client.put(
+#         url_for('csv.batch_modify_label', csv_id=csv_id),
+#         json={'changes': [{'context': 'column', 'index': 0, 'label': 'measurement'}]}
+#     )
+#     assert resp.status_code == 200
+#     assert (CSVFile.query.get(csv_id).measurement_table.columns ==
+#             ['header1', 'header2', 'header3']).all()
```

Comment:
It's okay to get rid of most of this.  There is fairly good coverage of the row/column related code at the model layer in `test_csv_file.py`.  I would replace it with one or two examples of non-trivial batch mutations.